lib/service/autoconfig/ispdb.php

```diff
@@ -36,7 +36,14 @@ public function __construct(Logger $logger) {
 
 	private function queryUrl($url) {
 		try {
-			$xml = @simplexml_load_file($url);
+			$content = file_get_contents($url, false, stream_context_create(array('http' => array('timeout' => 3))));
```

Comment:
Please make this multi-line and use the short array syntax like
```php
$content = file_get_contents($url, false, stream_context_create([
    'http' => [
        'timeout' => 3
    ]
]));
```

That fits our coding style better